src/github_runner_image_builder/config.py

```diff
@@ -102,6 +102,24 @@ class BaseImage(str, Enum):
     JAMMY = "jammy"
     NOBLE = "noble"
 
+    @classmethod
+    def get_version(cls, base: "BaseImage") -> Literal["22.04", "24.04"]:
+        """Change the codename to version tag.
+
+        Args:
+            base: The base image to get the version number for.
+
+        Return:
+            The release version of the current base image.
+        """
+        match base:
+            case BaseImage.JAMMY:
+                return "22.04"
+            case BaseImage.NOBLE:
+                return "24.04"
+        # This case is not possible
+        return "22.04"  # pragma: no cover
```

Comment:
Could we not remove this code and instead ignore another linting error (e.g. probably one for the typehint)?

Reply:
Would you mean we should remove this code and ignore the lint error?

Reply:
Removed!